src/framework/card/Card.jsx

```diff
@@ -59,12 +59,15 @@ export default class Card extends Component {
             </div>
           </div>
         </div>
+
+        {this.props.children}
```

Comment:
I find it somewhat weird that it's being passed a `children`. Even if it can be anything it feels misplaced as a ribbon... plus it 'only' takes an element, not like any or so. I think I'd done it as a `CardRibbon` and then being passed as a `ribbon` prop instead.